db/migrations/20230627112129_create_table_products.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE products (
+	id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(200) NOT NULL,
+	price DECIMAL NOT NULL,
+	category_id INT NOT NULL,
+	description TEXT NULL,
+	thumbnail_image VARCHAR(1000) NULL,
+	hover_image VARCHAR(1000) NULL,
+	detail_information TEXT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+	PRIMARY KEY (id)
```

Comment:
들여씌기 확인해 주세요.
FOREIGN KEY 빠진 부분 있습니다. 추가해 주세요.
```suggestion
CREATE TABLE products (
  id INT NOT NULL AUTO_INCREMENT,
  name VARCHAR(200) NOT NULL,
  price DECIMAL(10,2) NOT NULL,
  category_id INT NOT NULL,
  description TEXT NULL,
  thumbnail_image VARCHAR(1000) NULL,
  hover_image VARCHAR(1000) NULL,
  detail_information TEXT NULL,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (id)
```